### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by satyamvats5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
-# CS-01
+  CS-01
```
Comment (RudraNilBasu) [inline README.md]: Not needed. It is the markdown for the header of the Readme

File: README.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 
 ## Objectives
-
+   Our main objective is to implement the real life problems through computer science.
+  
 ## Course Structure
 
 ## Notifications
-
+   **Subscribe** to the mailing list of [Bytes Club](bytes-club@googlegroups.com)
```
Comment (RudraNilBasu) [inline README.md]: Link: https://groups.google.com/forum/#!forum/bytes-club (for joining request) or mailto:bytes-club@googlegroups.com (for mailing to the list)

File: README.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
+   in [issue-section](https://github.com/BytesClub/CS-01/issues).
+  
 ## Contribution
-
+   * In the specific problem-set make a directory of your user name which will contain
+    the required solution and make the PR.
+  
 ## Contact
+   * [Bytes club](https://bytesclub.github.io/) - <bytes-club@googlegroups.com>
+   * [Progyan Bhattacharya](codeprogyan.me) - <progyanb@acm.org>
```
Comment (RudraNilBasu) [inline README.md]: http://codeprogyan.me/

**Review by RudraNilBasu — CHANGES_REQUESTED** [review]

**Review by satyamvats5 — COMMENTED** [review]
is it ok?

**Pull request merged**